assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -45,9 +59,7 @@ const fetchGetModulesStore = createFetchStore( {
 		return {
 			...state,
 			isAwaitingModulesRefresh: false,
-			modules: modules.reduce( ( acc, module ) => {
-				return { ...acc, [ module.slug ]: module };
-			}, {} ),
+			modules: keyBy( sortBy( modules, [ ( { order } ) => order ] ), 'slug' ),
```

Comment:
Is this doing the same thing? I find this new "shorthand" much more difficult to read. The original one is standard ES6 and a common pattern in ES6 reducer code. The new one requires knowledge of the lodash API and—being significantly "denser" by virtual of being a one-liner, implicit return function—takes more effort to parse.

I'd rather this return to the previous implementation unless there's a runtime benefit to using lodash.

Reply:
I've reverted this back.